packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -217,8 +218,10 @@ export abstract class AbstractPureGraphManager {
   ): Promise<string>;
 
   // ------------------------------------------- Test  -------------------------------------------
+
   abstract runTests(
     graph: PureModel,
+    pureGraphManagerPlugins: PureGraphManagerPlugin[],
```

Comment:
Just remember that `AbstractPureGraphManager` has access to these plugins anyway, we should also swap order of `graph` and `testableInputs`